### PR TITLE
Expose InstallWebVRSpecShim

### PR DESCRIPTION
**Pull request by huningxin**
To fix https://github.com/toji/webvr.info/issues/63

### CODE REVIEW

**Comment by huningxin** [comment]
@jsantell @toji , PTAL. Thanks!

File: src/main.js [hunk 1]
```diff
@@ -89,3 +90,4 @@ if (!window.WebVRConfig.DEFER_INITIALIZATION) {
 }
 
 window.WebVRPolyfill = WebVRPolyfill;
+window.InstallWebVRSpecShim = InstallWebVRSpecShim;
```
Comment (cvan) [inline src/main.js]: can this instead be a method exposed as a method on `WebVRPolyfill`?

File: src/webvr-polyfill.js [hunk 2]
```diff
@@ -275,3 +275,4 @@ function InstallWebVRSpecShim() {
 WebVRPolyfill.version = version;
 
 module.exports.WebVRPolyfill = WebVRPolyfill;
+module.exports.InstallWebVRSpecShim = InstallWebVRSpecShim;
```
Comment (cvan) [inline src/webvr-polyfill.js]: same here

**Comment by cvan** [comment]
thanks for opening this PR. this is a good idea to expose the method. if you could add document this in the [`README.md`](https://github.com/googlevr/webvr-polyfill/blob/master/README.md#configuration) too, that'd be great.

**Comment by jsantell** [comment]
:+1: on attaching it to `WebVRPolyfill.installWebVRSpecShim` -- I don't think it makes sense as a prototype method, as instantiating the polyfill calls it anyway (and strange to have to manually call `WebVRPolyfill.prototype.installWebVRSpecShim()`.

File: src/main.js (hunk 1 above)
Comment (huningxin) [inline src/main.js]: sounds good. Will do that.

File: src/webvr-polyfill.js (hunk 2 above)
Comment (huningxin) [inline src/webvr-polyfill.js]: acked.

**Comment by huningxin** [comment]
Updated the PR according to your comments. @cvan @jsantell , please take another look.

**Comment by huningxin** [comment]
@cvan @jsantell , any comments?

**Comment by jsantell** [comment]
@huningxin  Looks good to me! Could you squash this down to one commit and I'll merge it in?

**Comment by toji** [comment]
LGTM as well. GitHub can do the squash and merge for us, any reason not to go that route?

**Comment by huningxin** [comment]
Thanks for review! And the commits have been squashed. Please help merge. Thanks!

**Comment by jsantell** [comment]
Thanks @huningxin! Merging in

**Pull request merged**